### PR TITLE
HTTPie 0.9.4

### PR DESCRIPTION
**Pull request by jkbrzt**
- Updates HTTPie to [v0.9.4](https://github.com/jkbrzt/httpie/releases/tag/0.9.4)
- Adds Python 3 as a dependency for better SSL support

### CODE REVIEW

**Comment by ilovezfs** [comment]
Test needs to use the current URL: https://raw.githubusercontent.com/Homebrew/homebrew-core/master/Formula/httpie.rb

**Comment by ilovezfs** [comment]
OK I fixed the test in https://github.com/Homebrew/homebrew-core/commit/43ea96c09378adcc9350088344b540eef1ea3c2e

**Comment by ilovezfs** [comment]
@BrewTestBot test this please

File: Formula/httpie.rb [hunk 1]
```diff
@@ -1,45 +1,44 @@
+# brew install https://gist.githubusercontent.com/jkbrzt/e59a0cde6ae6f59f2182/raw/httpie.rb
```
Comment (DomT4) [inline Formula/httpie.rb]: We don't need these two lines...

File: Formula/httpie.rb [hunk 2]
```diff
@@ -1,45 +1,44 @@
...
   desc "User-friendly cURL replacement (command-line HTTP client)"
   homepage "http://httpie.org/"
-  url "https://github.com/jkbrzt/httpie/archive/0.9.3.tar.gz"
-  sha256 "2a0c7cf6a6914620eebc2d7700e8e7a57aabde62bd62cd7fa68f8b216c0b2340"
+  url "https://pypi.python.org/packages/70/91/47c81b650d3cf33e7e9e639c315903947466a1993053fe1ff019f555ca97/httpie-0.9.4.tar.gz"
+  sha256 "0fc288a85d6c018c64bbc86dfcc9c7fad063e79816840dfa91e8d6c43654761e"
 
   head "https://github.com/jkbrzt/httpie.git"
 
-  bottle do
```
Comment (DomT4) [inline Formula/httpie.rb]: Please don't remove `bottle do` blocks on update. CI knows to ignore existing bottles.

File: Formula/httpie.rb [hunk 3]
```diff
@@ -1,45 +1,44 @@
...
-    cellar :any_skip_relocation
-    revision 1
-    sha256 "da9d2546404582dbc63b2143f71945321f4c3aacca794991f04f6e19f97a8493" => :el_capitan
-    sha256 "6008745c55d89aaf134ab8bec17f2188746149e9904f5b25f6a96d3cb878f784" => :yosemite
-    sha256 "3c3722eea4f74303af705f3bfb85c5c259f557af8689f36e6579a951f716b686" => :mavericks
-  end
-
-  depends_on :python if MacOS.version <= :snow_leopard
+  # The system OS X Python 2.7 is old and has poor SSL support.
```
Comment (DomT4) [inline Formula/httpie.rb]: Don't need either of these comments really.

File: Formula/httpie.rb [hunk 4]
```diff
@@ -1,45 +1,44 @@
...
   resource "requests" do
-    url "https://pypi.python.org/packages/source/r/requests/requests-2.9.1.tar.gz"
-    sha256 "c577815dd00f1394203fc44eb979724b098f88264a9ef898ee45b8e5e9cf587f"
+    url "https://pypi.python.org/packages/49/6f/183063f01aae1e025cf0130772b55848750a2f3a89bfa11b385b35d7329d/requests-2.10.0.tar.gz"
+    sha256 "63f1815788157130cee16a933b2ee184038e975f0017306d723ac326b5525b54"
   end
 
   def install
-    ENV.prepend_create_path "PYTHONPATH", libexec/"vendor/lib/python2.7/site-packages"
+    python = "python3"
```
Comment (DomT4) [inline Formula/httpie.rb]: If we're not using this more than once just inline it below.

File: Formula/httpie.rb [hunk 5]
```diff
@@ -1,45 +1,44 @@
...
 
-    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
-    system "python", *Language::Python.setup_install_args(libexec)
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python#{pyver}/site-packages"
+    system python, *Language::Python.setup_install_args(libexec)
 
-    bash_completion.install "httpie-completion.bash"
     bin.install Dir["#{libexec}/bin/*"]
     bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+
```
Comment (DomT4) [inline Formula/httpie.rb]: Don't need this empty line.

**Comment by ilovezfs** [comment]
@jkbrzt you need to use git rebase.

**Comment by jkbrzt** [comment]
I've opened a new pull request — #2572. Sorry about the mess!

**Pull request closed without merging**